tests/newman/files/small-files/replace-data/two.meta.csv

```diff
@@ -0,0 +1,6 @@
+col_name,col_type,label,indicator_grouping,indicator_unit,indicator_dp,filter_hint,filter_grouping_column
+ind_one,Indicator,Indicator one,,,,,
+filter_one,Filter,Filter one,,,,,Filter group one
```

Comment:
filter_grouping_column has to be the name of a column name in the data csv file or blank